navigation/los_guidance/launch/los_backstepping_simulator.launch

```diff
@@ -1,20 +1,16 @@
 <?xml version="1.0"?>
 <launch>
 
-<!--  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/> -->
-
   <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}" />
 
-  <rosparam command="load" file="$(find vortex)/config/robots/manta_enu.yaml"/>
+  <rosparam command="load" file="$(find auv_setup)/config/robots/manta_enu.yaml"/>
   <rosparam command="load" file="$(find thruster_interface)/config/thruster_terrapin.yaml" />
-  <rosparam command="load" file="$(find vortex)/config/environments/trondheim_freshwater.yaml"/>
+  <rosparam command="load" file="$(find auv_setup)/config/environments/trondheim_freshwater.yaml"/>
```

Comment:
These lines are also in the top lever launch file

Reply:
These launchfiles are actually unused, as the LOS package is launched directly from python scripts. For this reason I removed these launchfiles. Ref. e00be4c